packages/react/src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -296,13 +296,19 @@ export class FocusTrapZone extends React.Component<FocusTrapZoneProps, {}> {
       )
     }
 
-    // loop through all body's children, except the last one - which is the popup
-    for (let index = 0; index < bodyChildren.length - 1; index++) {
-      const element = bodyChildren[index]
-
-      if (element.getAttribute('aria-hidden') !== 'true') {
-        element.setAttribute('aria-hidden', 'true')
-        element.setAttribute(HIDDEN_FROM_ACC_TREE, 'true')
+    for (let index = 0; index < bodyChildren.length; index++) {
+      const currentChild = bodyChildren[index] as HTMLElement
+      const isOrHasFocusTrapZone =
+        currentChild === this._root.current || currentChild.contains(this._root.current)
+      const isAriaLiveRegion = currentChild.hasAttribute('aria-live')
```

Comment:
not a big deal, but no need to calculate this if `isOrHasFocusTrapZone` is false. you can just add it in the if isOrHasFocusTrapZone branch, get value through `getAttribute` and if that `!!<that-value>` is not true, do your setters.

Reply:
in case of `isOrHasFocusTrapZone` I don't need to read attribute, but yes, I can invert condition there, but then will need to rename it which I think can be less readable. The reason why I actually put all those calculation in variables is to have good readability of what is happening.
But I can do it for `isAriaLiveRegion` and change it to
```js
const isNotAriaLiveRegion = !currentChild.hasAttribute('aria-live')
```

Reply:
my vote for explicitness the code already has - this won't be an optimisation in any sense if we would skip an access to this attribute. Thus, if there won't be any alternative solution that would be on par in terms of expressiveness, I would prefer in a way it is now